samples/subsys/display/lvgl/sample.yaml

```diff
@@ -17,3 +17,5 @@ tests:
       - gui
     modules:
       - lvgl
+    integration_platforms:
+      - mimxrt1064_evk
```

Comment:
note: native_posix can be an option as well (with SDL display)

Reply:
yeah, tried this, but this requires 32bit SDL2 on the host and not everyone will have that.

Reply:
so maybe `native_posix_64`? `libsdl2-dev` seems to be a Zephyr requirement: https://docs.zephyrproject.org/latest/develop/getting_started/index.html, but... it probably does not belong to getting started...